kong.conf.default

```diff
@@ -48,6 +48,13 @@
 # NGINX
 #------------------------------------------------------------------------------
 
+#set_real_ip_from = 0.0.0.0/0;   # Defines trusted addresses that are known to send
+				 # correct replacement addresses. 
+				 # If the special value unix: is specified, 
+				 # all UNIX-domain sockets will be trusted.
+				 # See: http://nginx.org/en/docs/http/ngx_http_realip_module.html#set_real_ip_from
```

Comment:
Finally: please move this property *below* the `proxy_listen` and `proxy_listen_ssl` properties. And it should probably be renamed `proxy_set_real_ip_from`/.